backend/tournesol/serializers/inconsistencies.py

```diff
@@ -0,0 +1,50 @@
+from rest_framework import serializers
+from rest_framework.serializers import Serializer
+
+
+class ScoreInconsistencySerializer(Serializer):
+    """
+    Serializer for one element
+    """
+    inconsistency = serializers.FloatField()
+    criterion = serializers.CharField()
+    entity_1_uid = serializers.CharField()
+    entity_2_uid = serializers.CharField()
+    comparison_score = serializers.FloatField()
+    entity_1_rating = serializers.FloatField()
+    entity_2_rating = serializers.FloatField()
+
+
+class ScoreInconsistenciesStatsSerializer(Serializer):
+    """
+    Return some statistics for each criterion,
+    to be able to compare the results for the different criteria
+    """
+    mean_inconsistency = serializers.FloatField()
+    inconsistent_comparisons_count = serializers.IntegerField()
+    comparisons_count = serializers.IntegerField()
+
+
+class ScoreInconsistenciesSerializer(Serializer):
+    """
+    Returns the score inconsistencies and some statistics.
+    Used from a dictionary, not a queryset.
+    """
+    count = serializers.IntegerField()
+    results = ScoreInconsistencySerializer(many=True)
+    stats = serializers.DictField(child=ScoreInconsistenciesStatsSerializer(), label="criterion")
```

Comment:
DictFields are not used elsewhere. But it's quite practical for per-criterion stats.